src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -148,17 +169,34 @@ export const Interactive: StoryObj<Args> = {
   decorators: [
     (Story: Story) => (
       <div>
-        <p>
-          Click somewhere in this area to dismiss the tooltip, then hover over
-          the button to make it reappear.
-        </p>
+        <p>Hover over the button to make the tooltip appear.</p>
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export const InteractiveDisabled: StoryObj<Args> = {
+  render: () => (
+    <div
+      className={clsx(
+        styles['trigger--spacing-top'],
+        styles['trigger--spacing-left'],
+      )}
+    >
+      <Tooltip align="top" childDisabled={true} text={defaultArgs.text}>
+        <Button disabled variant="primary">
+          Tooltip trigger
+        </Button>
+      </Tooltip>
+    </div>
+  ),
+  decorators: [
+    (Story: Story) => (
+      <div>
+        <p>Hover over the button to make the tooltip appear.</p>
         <Story />
       </div>
     ),
   ],
-  play: async ({ canvasElement }) => {
-    const canvas = within(canvasElement);
-    const trigger = await canvas.findByRole('button');
-    await userEvent.hover(trigger);
-  },
```

Comment:
I removed the play method because it didn't seem to be used anywhere, unless it was so the tooltip appeared on page load?

Reply:
The intention was to capture the message being open in an interactive story in Chromatic but seeing as the stories are flaky, think we can remove the play method as mentioned. 

Can you change the button copy to `Tooltip trigger` or something relevant?

Reply:
You got it 👍